README.md

```diff
@@ -14,13 +11,35 @@ To initialize and seed the database with a sample scan, run `lein reset-db`.
 
 ### In docker
 
-Create a `docker-dev-resources/config.edn` file based on the sample
-file there. The `datomic` value is appropriate for running inside
-Docker.
-
-1. Set up [immutant-docker](https://github.com/turbovote/immutant-docker)
-as specified in its README.
-2. `./script/docker-run`
+Export your config environment variables:
+
+1. ENVIRONMENT
+    1. "dev" is a good starting point
+    1. this determines the lein profile to run under
```

Comment:
Should we do this? If we use the "dev" lein profile, it will no longer be using the resources/config.edn, and thus some of the settings below will not be honored.

I think we should ditch `ENVIRONMENT` and let the following vars control all of the configuration and not some changing subset of them.